apps/nextjs/tsconfig.json

```diff
@@ -15,13 +15,18 @@
     "noImplicitAny": false,
     "jsx": "preserve",
     /* Path Aliases */
-    "baseUrl": "./src",
+    "baseUrl": ".",
     "paths": {
-      "@/components/*": ["components/*"],
-      "@/app/*": ["app/*"],
-      "@/lib/*": ["lib/*"],
-      "@/utils/*": ["utils/*"],
-      "@/assets/*": ["assets/*"]
+      "@/components/*": ["src/components/*"],
+      "@/app/*": ["src/app/*"],
+      "@/lib/*": ["src/lib/*"],
+      "@/utils/*": ["src/utils/*"],
+      "@/assets/*": ["src/assets/*"],
+      "@/hooks/*": ["src/hooks/*"],
+      "ai-apps/*": ["src/ai-apps/*"],
```

Comment:
I'm seeing some strange behaviour where TS in VScode / tsc seems to resolve these differently, so import "hooks/..." doesn't work everywhere but import "../../hooks" does